internal/server/v1/page/golden/place_page/asm.Demographics.json

```diff
@@ -2600,12 +2600,12 @@
       }
     },
     "country/DEU": {
-      "date": "2021",
-      "value": 83196078,
+      "date": "2019",
+      "value": 83019213,
       "metadata": {
-        "importName": "WorldDevelopmentIndicators",
-        "provenanceUrl": "https://datacatalog.worldbank.org/dataset/world-development-indicators/",
-        "observationPeriod": "P1Y"
+        "importName": "EurostatData",
+        "provenanceUrl": "https://ec.europa.eu/eurostat/data/database",
+        "measurementMethod": "EurostatRegionalPopulationData"
```

Comment:
nit: Maybe we should prefer WDI over Eurostat?  I can send follow-up

https://github.com/datacommonsorg/mixer/blob/d7022933628a6df248842981d3cc75cbad697b8e/internal/server/ranking/ranking.go#L66-L74